RDKShell/RDKShell.h

```diff
@@ -28,6 +28,7 @@
 #include "tptimer.h"
 #ifdef ENABLE_RIALTO_FEATURE
 #include "RialtoConnector.h"
+#define RIALTO_TIMEOUT_MILLIS 2000
```

Comment:
Please set the timeout to 5000 millis. We noticed problems on realtek or amlogic platforms where 2000 was too small.

Reply:
Done